pkg/convert/tf.go

```diff
@@ -3234,6 +3249,21 @@ func translateModuleSourceCode(
 				}
 			}
 
+			if source, isLocal := moduleFromLocalFileSystem(moduleCall); isLocal {
+				key := filepath.Clean(filepath.Join(sourceDirectory, string(source)))
+				if _, seen := sandboxedModules[key]; !seen && wrapUsingTerraformModule {
+					state.sandboxedModuleNames[moduleCall.Name] = moduleName
+					sandboxedModules[key] = &PulumiTerraformModule{
+						packageName:     packageName,
+						moduleCall:      moduleCall,
+						local:           true,
+						absolutePath:    key,
```

Comment:
Curious is this truly absoultePath? filepath.Clean does not absolutize but perhaps sourceDirectory is absolute already.

Reply:
I _believe_ it is absolute because `sourceDirectory` should be absolute. Could add more tests in a follow up